src/bindings/python/simgrid_python.cpp

```diff
@@ -183,8 +425,10 @@ PYBIND11_MODULE(simgrid, m)
            "Retrieve a host by its name, or None if it does not exist in the platform.")
       .def_property_readonly("all_hosts", &Engine::get_all_hosts, "Returns the list of all hosts found in the platform")
       .def_property_readonly("all_links", &Engine::get_all_links, "Returns the list of all links found in the platform")
-      .def_property_readonly("all_netpoints", &Engine::get_all_netpoints, "Returns the list of all netpoints found in the platform")
-      .def_property_readonly("all_actors", &Engine::get_all_actors, "Returns the list of all actors found in the platform")
+      .def_property_readonly("all_netpoints", &Engine::get_all_netpoints,
+                             "Returns the list of all netpoints found in the platform")
+      .def_property_readonly("all_actors", &Engine::get_all_actors,
```

Comment:
It'd be better if you could not reindent the code that is not touched by your changes. Plus, you should read https://simgrid.frama.io/simgrid/Contributors_Documentation.html#enforcing-the-coding-standards